modules/ROOT/pages/debugging-with-toolbox.adoc

```diff
@@ -1,52 +1,42 @@
 = Debugging with Toolbx
 
-The FCOS image is kept minimal by design to reduce the image size and the
-attack surface. This means that it does not include every troubleshooting tools
-that a normal OS may include. Instead, the recommended approach is to leverage
-containers with the https://containertoolbx.org/[toolbox] utility
-included in the image.
+The FCOS image is kept minimal by design to reduce the image size and the attack surface.
+This means that it does not include every troubleshooting tools that a normal OS may include.
+Instead, the recommended approach is to leverage containers with the https://containertoolbx.org/[toolbox] utility included in the image.
 
 == What is Toolbx?
 
-Toolbx is a utility that allows you to create privileged containers meant to
-debug and troubleshoot your instance. It is a wrapper around podman which
-starts long running containers with default mounts and namespaces to facilitate
-debugging the host system.
+Toolbx is a utility that allows you to create privileged containers meant to debug and troubleshoot your instance.
+It is a wrapper around podman which starts long running containers with default mounts and namespaces to facilitate debugging the host system.
 
-These containers can then be used to install tools that you may need for
-troubleshooting.
+These containers can then be used to install tools that you may need for troubleshooting.
 
 == Using Toolbx
 
-You can create a new toolbox by running the command below. On the first run,
-this will ask you if you want to use the image
-`registry.fedoraproject.org/fedora-toolbox:35`.
+You can create a new toolbox by running the command below.
```

Comment:
I would keep some of what was there:

```suggestion
You can create a new toolbox by running the command below. On the first run it will ask you if you want to download an image. Answer yes with `y`. 
```